swift/extractor/translators/DeclTranslator.h

```diff
@@ -70,9 +70,11 @@ class DeclTranslator : public AstTranslatorBase<DeclTranslator> {
     std::optional<TrapClassOf<D>> entry;
     auto id = dispatcher.assignNewLabel(decl, mangledName(decl));
     if (dispatcher.shouldEmitDeclBody(decl)) {
+      dispatcher.extractedDeclaration(&decl);
       entry.emplace(id);
       fillDecl(decl, *entry);
     }
+    dispatcher.skippedDeclaration(&decl);
```

Comment:
shouldn't this be in an `else` clause? There's no early return before this

Reply:
Good catch!